crates/executor/src/blockchain_tree/block_indices.rs

```diff
@@ -0,0 +1,258 @@
+//! Implementation of [`BlockIndices`] related to [`super::BlockchainTree`]
+
+use super::chain::{Chain, ChainId, ForkBlock};
+use reth_primitives::{BlockHash, BlockNumber, SealedBlockWithSenders};
+use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
+
+/// Internal indices of the block.
```

Comment:
As a first time reader I don't really know what indices refers to here, so I am sort of surprised to see it contain *a lot* of data, more expanded documentation here would be helpful

Reply:
Will do that